src/modules/shared/components/PaginationBar/PaginationBar.tsx

```diff
@@ -47,18 +48,26 @@ const PaginationBar: FC<PaginationBarProps> = ({
 	);
 
 	return (
-		<div className={clsx(className, styles['c-pagination-bar'])}>
+		<div
+			className={clsx(
+				className,
+				styles['c-pagination-bar'],
+				showBackToTop && styles['c-pagination-bar--back-to-top']
+			)}
+		>
 			{renderProgress()}
 
 			{renderPagination()}
 
-			<Button
-				// TODO: smooth scroll to top
-				className={styles['c-pagination-bar__back-to-top']}
-				variants={['text', 'neutral']}
-				label="Terug naar boven"
-				iconEnd={<Icon name="arrow-up" />}
-			/>
+			{showBackToTop && (
+				<Button
+					// TODO: smooth scroll to top
```

Comment:
Moet deze nog opgepakt worden?

Reply:
Ik heb die TODO al gefixed in een andere PR. Dan was die functionaliteit al werkende op de demo.